src/enemySpawn.js

```diff
@@ -0,0 +1,20 @@
+import Enemy from './enemy';
+
+function enemySpawn() {
+    const enemies = [];
+    const enemiesCount = 3;
+    const yPos = [62.5, 175, 287.5];
+    const gameBoard = document.getElementById('game-board');
```

Comment:
gameboard can be imported from gameBoard.js and its element property can be used to reference the html element